{"name": "test_forgot_account", "status": "failed", "statusDetails": {"message": "AssertionError: Expected 'Error!\n  The customer information provided could not be found.' but got: 'Customer Lookup \n  Your login information was located successfully. You are now logged in.\n   Username: parasoft\n   Password: demo'\nassert 'Error!\\nThe ...not be found.' == 'Customer Loo...assword: demo'\n  \n  + Error!\n  + The customer information provided could not be found.\n  - Customer Lookup \n  - Your login information was located successfully. You are now logged in.\n  -  Username: parasoft\n  -  Password: demo", "trace": "browserInstance = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"7a77c1fac652576ef4fa66c0ca8d4b94\")>\n\n    @pytest.mark.smoke\n    def test_forgot_account(browserInstance):\n        driver = browserInstance\n        driver.get(\"https://parabank.parasoft.com/parabank/index.htm\")\n        browserInstance.maximize_window()\n        user_forgot_network = forgot_account(browserInstance)\n        user_forgot_network.click_on_forgot()\n        user_forgot_network.first_name(user_data[\"user_first_name\"])\n        user_forgot_network.last_name(user_data[\"user_second_name\"])\n        user_forgot_network.forgot_address(user_data[\"user_address\"])\n        user_forgot_network.city(user_data[\"user_city\"])\n        user_forgot_network.forgot_state(user_data[\"user_state\"])\n        user_forgot_network.zip_code(user_data[\"user_zip_code\"])\n        user_forgot_network.forgot_ssn(user_data[\"user_ssn\"])\n        user_forgot_network.forgot_button()\n        time.sleep(3)\n        compare = user_forgot_network.right_panel()\n        welcome_message = f\"Customer Lookup \\nYour login information was located successfully. You are now logged in.\\n Username: parasoft\\n Password: demo\"\n        print(compare)\n    \n>       assert compare.strip() == welcome_message.strip(), \\\n                f\"Expected '{compare}' but got: '{welcome_message}'\"\nE       AssertionError: Expected 'Error!\nE         The customer information provided could not be found.' but got: 'Customer Lookup \nE         Your login information was located successfully. You are now logged in.\nE          Username: parasoft\nE          Password: demo'\nE       assert 'Error!\\nThe ...not be found.' == 'Customer Loo...assword: demo'\nE         \nE         + Error!\nE         + The customer information provided could not be found.\nE         - Customer Lookup \nE         - Your login information was located successfully. You are now logged in.\nE         -  Username: parasoft\nE         -  Password: demo\n\nTests\\test_forgot_account.py:36: AssertionError"}, "attachments": [{"name": "screenshot_on_failure", "source": "05aa59fc-9aec-4f1f-8f0b-ea57774986af-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "ce8df995-46ba-48f1-80e0-48990c4dd12f-attachment.txt", "type": "text/plain"}], "start": 1758269072699, "stop": 1758269085311, "uuid": "fd8b4b37-a6f4-4a84-a3f8-d1709bb10896", "historyId": "94e38a5b2a8cdb323e4ceb63d96945e8", "testCaseId": "94e38a5b2a8cdb323e4ceb63d96945e8", "fullName": "Tests.test_forgot_account#test_forgot_account", "labels": [{"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_forgot_account"}, {"name": "host", "value": "MSI"}, {"name": "thread", "value": "8508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_forgot_account"}], "titlePath": ["Tests", "test_forgot_account.py"]}